docs/getting-started/overview.rst

```diff
@@ -1,12 +1,33 @@
 .. _overview:
 
-Overview
-========
+What is Scipp?
+==============
+
+Scipp is the result of a long evolution of packages for scientific computing in Python.
+`NumPy <https://numpy.org/>`_, the fundamental package for scientific computing in Python providing a multidimensional array object, sits at the core of this evolution.
+On top of this libraries such as `SciPy <https://scipy.org/>`_, `Pandas <https://pandas.pydata.org/>`, and `Xarray <https://docs.xarray.dev>`_ provide more advanced functionality and user convenience.
+
+Scipp's central ambition is to make scientific computing **safe** and **intuitive**.
+Safety, i.e., correctness of scientific results, is crucial given the rapidly growing complexity of scientific data and data analysis.
+At the same time it has to be ensured that scientific results are reproducible.
+Open data alone is not sufficient if it is analysed using "black-box"-like applications that do not let the user, referee, or other scientists re-run the analysis or inspect and understand each processing step.
+
+Scipp aims to achieve its ambition in multiple ways:
+
+- A concise and intuitive "language" (programming interface) allows the user to clearly express their intent, and makes this intent apparent to future readers of the code.
+  For example, *named dimensions* remove the need for cryptic axis indices or explicit array transposition and broadcasting.
+- Coordinate arrays associated with and array of data ensure that no incompatible data is combined and allow for direct creation of meaning plots with labelled axis tick marks.
+- Physical units associated with all arrays as well as their coordinate arrays eliminate a whole set of potential errors.
+  This includes the risk of combining data with different units (such as adding "meters" and "seconds") or unit scales (such as adding "meters" and "millimeters").
+- Features targeting `Jupyter <https://jupyter.org/>`_, such as rich HTML visualizations of data, give the user the tools to understand their data at every step and share their work with collaborators.
+
+Scipp's core is written in C++ with multi-threading enabled by default, providing good out-of-the-box performance for many applications.
+
 
 The data array concept
 ----------------------
 
-The core data structure of scipp is :py:class:`scipp.DataArray`.
+The core data structure of Scipp is :py:class:`scipp.DataArray`.
 A data array is essentially a multi-dimensional array with associated dicts of coordinates, masks, and attributes.
```

Comment:
All your remaining comments to this file address existing, essentially unchanged content (apart from a moved paragraph). Since we were unsure if it should be kept, or maybe combined with the "Quick Start" page, or other, I suggest to not address these here.

Reply:
I thought you wanted to make that decision here. But we can defer it until later.

Reply:
I'd like to split it, since it may require more work to update other pages, or merge/rewrite things.